fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/cursors/SizeStatisticsCollectorCursor.java

```diff
@@ -217,32 +222,30 @@ private SizeStatisticsCollectorCursorContinuation(RecordCursorResult<KeyValue> c
                     return RecordCursorProto.SizeStatisticsContinuation.newBuilder().setPartialResults(this.sizeStatisticsResults.toProto()).setContinuation(ByteString.copyFrom(b)).build();
                 } else {
                     //nothing more to aggregate
-                    return RecordCursorProto.SizeStatisticsContinuation.newBuilder().setContinuation(ByteString.copyFrom(RecordCursorEndContinuation.END.toBytes())).build();
+                    return RecordCursorProto.SizeStatisticsContinuation.newBuilder().build();
```

Comment:
```suggestion
                    return RecordCursorProto.SizeStatisticsContinuation.getDefaultInstance();
```

Possibly unnecessary, but it stops it from creating an unnecessary object.